Wrapper function

```diff
@@ -0,0 +1,19 @@
+# Input files
```

Comment:
Could you please move this file under a folder, e.g., `RWrapper`, and rename the file by removing the space between `Wrapper` and `function` or replacing it with `_`?  What is the extension of this file?

Reply:
It's done, thank you! 
I have specified the extension of the file (.R ).